web/src/test/java/com/epam/rft/atsy/cucumber/util/CommonStepDefs.java

```diff
@@ -2,12 +2,23 @@
 
 import static com.epam.rft.atsy.cucumber.util.DriverProvider.getDriver;
 import static com.epam.rft.atsy.cucumber.util.DriverProvider.waitForPageLoadAfter;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
 
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
 import org.openqa.selenium.By;
 
 import cucumber.api.java.en.Given;
+import org.openqa.selenium.WebElement;
 
 public class CommonStepDefs {
+  private DriverProvider driverProvider;
+
+  public CommonStepDefs(DriverProvider driverProvider) {
```

Comment:
Please remove the `driverProvider` field and the added constructor as the `getDriver()` method call is the same as the `driverProvider.getDriver()` call.